### DISCUSSION TITLE
How to upgrade photoprism

### DISCUSSION THREAD

**Discussion by jaydhary14** [tteck/Proxmox#1468] [General]
Trying to figure out how to upgrade photoprism, please help

**Comment by tteck** [comment]
Have you attempted to follow the instructions?

![Screenshot 2023-06-08 9 48 10 AM](https://github.com/tteck/Proxmox/assets/21342670/4cc2c572-db7e-4c0c-8603-a4fed846bfc0)

**Reply by CoreyJ87** [reply]
@tteck does the your LXC update script not upgrade the applications? I was under the impression that it ran the "update_script" part of the install script

**Reply by tteck** [reply]
No, only the OS. Look at the above post.

**Reply by CoreyJ87** [reply]
I get that I just thought that the update script did that as well. Good to know. Thanks

**Reply by villamanuel1980** [reply]
> Have you attempted to follow the instructions?
> 
> ![Screenshot 2023-06-08 9 48 10 AM](https://private-user-images.githubusercontent.com/21342670/244401450-4cc2c572-db7e-4c0c-8603-a4fed846bfc0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4MDc0MzYsIm5iZiI6MTcxODgwNzEzNiwicGF0aCI6Ii8yMTM0MjY3MC8yNDQ0MDE0NTAtNGNjMmM1NzItZGI3ZS00YzBjLTg2MDMtYTRmZWQ4NDZiZmMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDE0MjUzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFjY2NmY2U5MTlkM2ZlMjEyNGFhYmM1NmYyOGZjYWUzMjA2NDE1NmI2N2JiNTFlMmJlN2RiNzVhYTMxMjlhNDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ayWsJU_jXdisEd_b5vJKg_17LjJOhqn4diXIlZnXpOo)

Hello! I did this and got this error message on two separate instances of Photoprism I'm running.. Can anybody help? Thanks!!

[ERROR] in line 67: exit code 0: while executing command apt-get install -y libvips42 &> /dev/null

**Reply by villamanuel1980** [reply]
> > Have you attempted to follow the instructions?
> > ![Screenshot 2023-06-08 9 48 10 AM](https://private-user-images.githubusercontent.com/21342670/244401450-4cc2c572-db7e-4c0c-8603-a4fed846bfc0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4MDc0MzYsIm5iZiI6MTcxODgwNzEzNiwicGF0aCI6Ii8yMTM0MjY3MC8yNDQ0MDE0NTAtNGNjMmM1NzItZGI3ZS00YzBjLTg2MDMtYTRmZWQ4NDZiZmMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDE0MjUzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFjY2NmY2U5MTlkM2ZlMjEyNGFhYmM1NmYyOGZjYWUzMjA2NDE1NmI2N2JiNTFlMmJlN2RiNzVhYTMxMjlhNDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ayWsJU_jXdisEd_b5vJKg_17LjJOhqn4diXIlZnXpOo)
> 
> Hello! I did this and got this error message on two separate instances of Photoprism I'm running.. Can anybody help? Thanks!!
> 
> [ERROR] in line 67: exit code 0: while executing command apt-get install -y libvips42 &> /dev/null

Updating after some tinkering, this seems to be the issue. I have no idea how to update GLIBC inside the LXC or how to move my database to a new instance of photoprism.. :(

Jun 19 16:54:38 photoprism photoprism[138]: /opt/photoprism/bin/photoprism: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /opt/p>
Jun 19 16:54:38 photoprism photoprism[138]: /opt/photoprism/bin/photoprism: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /opt/p>
Jun 19 16:54:38 photoprism photoprism[138]: /opt/photoprism/bin/photoprism: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /opt/p>
Jun 19 16:54:38 photoprism systemd[1]: photoprism.service: Control process exited, code=exited, status=1/FAILURE
Jun 19 16:54:38 photoprism systemd[1]: photoprism.service: Failed with result 'exit-code'.

Can someone give me a hint? Thanks!!

**Comment by jaydhary14** [comment]
Oh wow , sorry did not notice that. Give that a try. You are awesome btw thanks

From: tteckster ***@***.***>
Date: Thursday, June 8, 2023 at 8:51 AM
To: tteck/Proxmox ***@***.***>
Cc: jaydhary14 ***@***.***>, Author ***@***.***>
Subject: Re: [tteck/Proxmox] How to upgrade photoprism (Discussion #1468)

Have you attempted to follow the instructions?

[Image removed by sender. Screenshot 2023-06-08 9 48 10 AM]<https://user-images.githubusercontent.com/21342670/244401450-4cc2c572-db7e-4c0c-8603-a4fed846bfc0.png>

**Comment by jaydhary14** [comment]
I just tested it and worked perfectly, thanks

From: Jay Choudhary ***@***.***>
Date: Thursday, June 8, 2023 at 8:59 AM
To: tteck/Proxmox ***@***.***>, tteck/Proxmox ***@***.***>
Cc: Author ***@***.***>
Subject: Re: [tteck/Proxmox] How to upgrade photoprism (Discussion #1468)
Oh wow , sorry did not notice that. Give that a try. You are awesome btw thanks

From: tteckster ***@***.***>
Date: Thursday, June 8, 2023 at 8:51 AM
To: tteck/Proxmox ***@***.***>
Cc: jaydhary14 ***@***.***>, Author ***@***.***>
Subject: Re: [tteck/Proxmox] How to upgrade photoprism (Discussion #1468)

Have you attempted to follow the instructions?

[Image removed by sender. Screenshot 2023-06-08 9 48 10 AM]<https://user-images.githubusercontent.com/21342670/244401450-4cc2c572-db7e-4c0c-8603-a4fed846bfc0.png>
